contracts/test/TestSegmentUtils.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity 0.4.24;
+pragma experimental "v0.5.0";
+
+import "../utils/SegmentUtils.sol";
+
+contract TestSegmentUtils {
```

Comment:
Why this is a `contract`, not `library`?

Reply:
Ok, I see this is just a test, so keeping state variables is ok for it. I suggest moving this file to the mocks folder instead of `test`. Or we can create a full solidity test from it and move to tests folder.